### ISSUE TITLE
Limit amount of author names shown in library entries.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#9794]
### Describe your suggested feature

Basically so that the below doesn't happen.
![Screenshot_20230803_020323_Tachiyomi](https://github.com/tachiyomiorg/tachiyomi/assets/63903377/0c47720d-c862-409f-9d5b-db2488c0616d)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
This is a bit of an edge case, and there'd be no other way to view that info.

**Comment by THElegend5** [comment]
> This is a bit of an edge case, and there'd be no other way to view that info.

Yeah this is kinda an edge case but I have quite a few that take up half the screen.
I was thinking maybe something like three dots at the end of author names which can be used to expand them back could be added so that they are still accessible ?

**Comment by THElegend5** [comment]
@arkon would you consider something like the above?

**Comment by Sardonyx001** [comment]
Could you provide us with more information about how to reproduce this? Like which extension did you use etc.

**Comment by THElegend5** [comment]
> Could you provide us with more information about how to reproduce this? Like which extension did you use etc. 

Can't really be reproduced as this is from my instance of komga which is self hosted. But is basically caused by an entry having a ton of authors which in this case was a result of it's long run and a lot of author changes.

**Comment by Sardonyx001** [comment]
I'm checked out exactly how the code that renders the author's info section is laid out and if I can't reproduce it I can't really test any fixes... I've tried to look for entries with long author tags but had very little success as many sources just provide "Various" as the author.

**Comment by THElegend5** [comment]
> I'm checked out exactly how the code that renders the author's info section is laid out and if I can't reproduce it I can't really test any fixes... I've tried to look for entries with long author tags but had very little success as many sources just provide "Various" as the author. 

@Sardonyx001 Here you go found one.
https://mangadex.org/title/7895cbd9-2ce9-4826-823f-6811fef7179e/though-i-may-be-a-villainess-i-ll-show-you-i-can-obtain-happiness
![Screenshot_20230806_230558_Tachiyomi.jpg](https://github.com/tachiyomiorg/tachiyomi/assets/63903377/c55614ad-ecf5-4146-92bd-e04abe7319f1)

**Comment by arkon** [comment]
You can always temporarily hardcode something to test.

**Comment by Sardonyx001** [comment]
@arkon Totally didn't think of that thank you for the suggestion! 
@THElegend5 Thanks! I'll give this a try and get back to you. 
I may need to create some ExpandableText class for this to work but I really want to make this as minimal as possible...
